scene/main/canvas_item.cpp

```diff
@@ -445,6 +503,42 @@ int CanvasItem::get_light_mask() const {
 	return light_mask;
 }
 
+void CanvasItem::set_layer_mask(int p_layer_mask) {
```

Comment:
I would most likely name this visibility_layer to avoid confusing with the physics.

Reply:
I copied as much as I could from VisualInstance3D to keep it consistent, `VisualInstance3D::set_layer_mask`. Should it diverge?

Reply:
With the removals in a17fdaef09f686f7c61b7f25439061de99a863c0 is this still an issue? Physics seems to all start with `collision_` now (071871b787684d03fd924f69603107860c8bdb99).

I'd prefer to match `VisualInstance3D` and rename if it got the same treatment.

Reply:
I think VisualInstance3D should also be renamed to this in a separate PR.

Reply:
But that said, in the 3D side, physics and rendering are separate nodes, but this is not the case of 2D, where all 2D nodes can render.